docs/superpowers/plans/2026-05-12-agent-sandbox.md

```
@@ -0,0 +1,2521 @@
+**Files:**
+- Create: `docs/superpowers/specs/2026-05-12-agent-sandbox-design.poc-report.md`
+
+- [ ] **Step 1: 跑 PoC #1（自动化）**
+
+Run: `uv run python scripts/dev/sandbox_poc.py > /tmp/poc1.json`
+Expected: stdout 含 `"passed": true` 或 `false`；记录结果
+
+- [ ] **Step 2: 手动跑 PoC #2 — 敏感文件读拒**
+
+启动开发服务器：
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)